src/dune_rules/workspace.ml

```diff
@@ -37,6 +39,7 @@ module Lock_dir = struct
             (List.map repositories ~f:snd) )
       ; "constraints", Dyn.list Dune_lang.Package_dependency.to_dyn constraints
       ; "pins", (Dyn.list Dyn.string) (List.map pins ~f:snd)
+      ; "constraints", (Dyn.list Package.Name.to_dyn) (List.map ~f:snd depopts)
```

Comment:
I assume this is a copy-paste oversight:

```suggestion
      ; "depopts", (Dyn.list Package.Name.to_dyn) (List.map ~f:snd depopts)
```

Reply:
Indeed it was